onnxruntime/core/optimizer/skip_layer_norm_fusion.cc

```diff
@@ -230,18 +234,18 @@ Status SkipLayerNormFusion::ApplyImpl(Graph& graph, bool& modified, int graph_le
                                                "SkipLayerNormalization",
                                                "fused SkipLayerNorm subgraphs ",
                                                skip_layer_norm_input_defs,
-                                               {}, {}, kMSDomain);
+                                               ln_node.MutableOutputDefs(), {}, kMSDomain);
```

Comment:
Why not use graph_utils::MoveAllNodeOutputs to move outputs (including edges)?

Reply:
Actually MoveAllNodeOutputs is useless. All the edge will be re-built in graph.resolve